assets/templates/dataset-filter/filter-overview.tmpl

```diff
@@ -19,12 +19,12 @@
                   {{ template "partials/latest-release-alert" . }}
 	            {{ end }}
                     <div class="col col--md-47 col--lg-59 margin-top--2 margin-bottom--4 background--gallery">
-                        <ul class="list--neutral filter-overview">
-                            <li class="margin-left--0 padding-bottom--2 padding-top--0 padding-right--2 width-lg--56">
+                        <ul class="list--neutral filter-overview ">
+                            <li class="font-size--18 line-height--32 margin-left--0 padding-bottom--2 padding-top--0 padding-right--2 width-lg--56">
                         	    <a class="float-el--right-md float-el--right-sm float-el--right-lg" href="{{.Data.ClearAll.URL}}">Clear filters</a>
                             </li>
                             {{ range .Data.Dimensions}}
-                            <li class="js-filter-option white-background margin-left-md--2 margin-right-md--2 margin-right-sm--1 margin-left-sm--1 {{if eq .Link.Label "Add"}}filter-overview__add{{else}}filter-overview__edit{{end}}">
+                            <li class="font-size--18 line-height--32 js-filter-option white-background margin-left-md--2 margin-right-md--2 margin-right-sm--1 margin-left-sm--1 {{if eq .Link.Label "Add"}}filter-overview__add{{else}}filter-overview__edit{{end}}">
```

Comment:
With `line-height--32`, there is a slight misalignment between the Add button and the dimension name/"Nothing added" text. We may want to check whether we need the line-height set here, or if the button needs to be updated to align with the text

![image](https://user-images.githubusercontent.com/23668262/101457139-08454100-392d-11eb-8730-5ef699c8766e.png)